website/src/pages/api/auth/[...nextauth].ts

```diff
@@ -32,6 +34,23 @@ if (process.env.DISCORD_CLIENT_ID) {
   );
 }
 
+if (boolean(process.env.DEBUG_FAKE_USERS)) {
```

Comment:
Should we just do this whenever we're running in development mode? If so we could cut out a new env variable and make this

```
if (process.env.NODE_ENV === "development") {
```

Reply:
that's a great idea. I've pushed a change for this and adjusted documentation